### DISCUSSION TITLE
Paging backwards with cursor-based pagination

### DISCUSSION THREAD

**Discussion by ericchernuka** [mswjs/data#247] [Ideas]
Wondering if there is any appetite for `take` using a negative number to allow for paging backward easily? I haven't dug into the code to see the implications of a change like this but I wanted to get some feedback since it seems incredibly useful and straightforward for users.

https://www.prisma.io/docs/concepts/components/prisma-client/pagination#example-paging-backwards-with-cursor-based-pagination

**Comment by kettanaito** [comment]
Backward pagination is now supported since v1.1.0!

https://github.com/mswjs/data/blob/934b168192e4b31bfdc620ebf4c5fdc4b178438d/tests/pagination-offset.test.ts#L164-L168

Also possible with cursor-based pagination:

https://github.com/mswjs/data/blob/934b168192e4b31bfdc620ebf4c5fdc4b178438d/tests/pagination-cursor.test.ts#L89-L94
